features/steps/thens/attributes.py

```diff
@@ -73,3 +73,31 @@ def _():
     misc.handle_errors(context, list(_()))
 
 
+@then('The type of attribute {attribute} must be {value}')
```

Comment:
Discussion needed: how is this function different from the one in line 64-73? What is the difference between 'must' and 'should', if any in this context?